atc/db/team.go

```diff
@@ -545,7 +545,7 @@ func (t *team) PublicPipelines() ([]Pipeline, error) {
 			"team_id": t.id,
 			"public":  true,
 		}).
-		OrderBy("team_id ASC", "ordering ASC").
+		OrderBy("t.name ASC", "ordering ASC").
```

Comment:
Same here! Not sure if this is used for the ordering?

Reply:
`PublicPipelines()` is called by `pipelineserver`, which lists exposed pipelines to un-logged-in users. This changes makes teams with public pipelines also ordered by team name.